### DISCUSSION TITLE
Hosting & MKDocs

### DISCUSSION THREAD

**Discussion by Slluxx** [obhq/obliteration#843] [General]
Stumbled across this on accident but i have to say that this doesnt look professional at all. I wouldn't trust a software where a PR escalates into a discussion that should be held in private. (The moment i wanted to hit send, the PR was locked)

Aside from that, i dont even understand the need for "real" web hosting. MKDocs (great choice and good alternative for GH limited wiki) aswell as any other website you need up, can be hosted via Github Pages and is **not hackable**, due it simply being a static site. There is no need to pay any money either (except if you want your own domain name).

If you want dynamic-ish content (for example when issues or PRs are made) you can use github actions and build anything you could possibly want - no need for any kind of server database.

Just my two cents

**Comment by ultimaweapon** [comment]
> Stumbled across this on accident but i have to say that this doesnt look professional at all. I wouldn't trust a software where a PR escalates into a discussion that should be held in private. (The moment i wanted to hit send, the PR was locked)

I'm sorry for that. I'm not in the Discord so GitHub is the only place to discus with me.

**Comment by SoffiMeower** [comment]
Yeah... If we could've done this privately, we would've... But the moment Ultima tried to add me on discord so he could join the server, he kinda got auto-flagged and banned
